egs/aishell/s10/local/run_chain.sh

```diff
@@ -191,33 +193,35 @@ if [ $stage -le 16 ]; then
     ${dir}/processed_egs ${dir}/egs
 fi
 
-if [[ $stage -le 17 ]]; then
-  echo $dir
-  mkdir -p $dir/configs
-  if $train_ivector; then
-    cat <<EOF > $dir/configs/init.config
+lda_mat_filename=
+if [[ "$feat_type" == "lda" ]]; then
+  if [[ $stage -le 17 ]]; then
+    echo $dir
+    mkdir -p $dir/configs
+    if $train_ivector; then
+      cat <<EOF > $dir/configs/init.config
 input-node name=ivector dim=100
-input-node name=input dim=40
-output-node name=output input=Append(Offset(input, -1), input, Offset(input, 1), ReplaceIndex(ivector, t, 0))
+input-node name=input dim=120
```

Comment:
Is it possible to read 120 from feats.scp?
E.g., 3 x `feat-dim`

Reply:
Done